core/server/services/auth/session/middleware.js

```diff
@@ -1,4 +1,4 @@
-const {URL} = require('url');
+const {parse: parseUrl} = require('url');
```

Comment:
We try to use a package notation in Ghost.
e.g. lookup `url.parse` in the code base